### PR TITLE
CLDR-15484 Fix mistakes

### PR DESCRIPTION
**Pull request by ivan-pan**
CLDR-15484

- [ ] This PR completes the ticket.

### CODE REVIEW

File: docs/ldml/tr35-general.md [hunk 1]
```diff
@@ -1303,7 +1303,7 @@ The durationUnit is a special type of unit used for composed time unit durations
 </durationUnit>
 ```
 
-The type contains a skeleton, where 'h' stands for hours, 'm' for minutes, and 's' for sections. These are the same symbols used in availableFormats, except that there is no need to distinguish different forms of the hour.
+The type contains a skeleton, where 'h' stands for hours, 'm' for minutes, and 's' for seconds. These are the same symbols used in availableFormats, except that there is no need to distinguish different forms of the hour.
```
Comment (srl295) [inline docs/ldml/tr35-general.md:1306]: thanks for catching these…

**Review by srl295 — DISMISSED** [review]
Great finds.

**Comment by srl295** [comment]
@ivan-pan can you rebase?

**Comment by ivan-pan** [comment]
I noticed Гриинуич in Part 3. This should be changed to [Гринуич](https://bg.wikipedia.org/wiki/%D0%93%D1%80%D0%B8%D0%BD%D1%83%D0%B8%D1%87) in the locale data and, when done, in the LDML specification.

**Comment by ivan-pan** [comment]
On another note, the HTML version of [Part 5](https://unicode-org.github.io/cldr/ldml/tr35-collation.html) contains `:point_right:` instead of `👉`.

**Comment by ivan-pan** [comment]
In Part 6, something is wrong here:

> \* The type attribute contained only a `unicode_region_subtag` `unicode_subdivision_suffix` values were used in the `contains` attribute; these are not unique across multiple territories, so for lower levels a now-deprecated

> Therer mayreflecting that _force per area_ can be referenced as either _pressure_ or _stress_, for example).

**Review by srl295 — APPROVED** [review]

**Comment by srl295** [comment]
> In Part 6, something is wrong here:
> 
> > * The type attribute contained only a `unicode_region_subtag` `unicode_subdivision_suffix` values were used in the `contains` attribute; these are not unique across multiple territories, so for lower levels a now-deprecated
> 
> > Therer mayreflecting that _force per area_ can be referenced as either _pressure_ or _stress_, for example).

First, thanks once again! Second, I'll take a look at this.

**Comment by ivan-pan** [comment]
@srl295 Please also take a look at these two comments of mine:

> I noticed Гриинуич in Part 4. This should be changed to [Гринуич](https://bg.wikipedia.org/wiki/%D0%93%D1%80%D0%B8%D0%BD%D1%83%D0%B8%D1%87) in the locale data and, when done, in the LDML specification.

> On another note, the HTML version of [Part 5](https://unicode-org.github.io/cldr/ldml/tr35-collation.html) contains `:point_right:` instead of `👉`.

**Comment by srl295** [comment]
> I noticed Гриинуич in Part 4. This should be changed to [Гринуич](https://bg.wikipedia.org/wiki/%D0%93%D1%80%D0%B8%D0%BD%D1%83%D0%B8%D1%87) in the locale data and, when done, in the LDML specification.

Can you file a separate ticket?

**Comment by ivan-pan** [comment]
@srl295 Oh, it actually is already corrected in the current CLDR version, now it needs to be corrected in the specification.

**Pull request merged**